styles.css

```diff
@@ -0,0 +1,3 @@
+.my-greeting {
+    background-color: cyan;
```

Comment:
[OPTIONAL] It is considered bad practice to use color names. Consider changing it to its `hex` or `rgb` equivalent